apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Genesis.java

```diff
@@ -94,6 +94,7 @@ static void apply() {
         if (genesisAccountsJSON == null) {
             loadGenesisAccountsJSON();
         }
+        CDI.current().select(BlockchainConfig.class).get().reset();
```

Comment:
It seems like that is not needed ???
private static BlockchainConfig blockchainConfig........

Reply:
CDI.current().select... construction is rather for fixing "bad" behavior, but not for "everyday use".